### ISSUE TITLE
Reset playback status

### ISSUE THREAD

**Issue by billystar1337** [razzeee/script.trakt#453]
hallo everyone,

since the update to kodi 18 i have a problem. i sync my whole library and everythings fine so far.
but in a older version of kodi i started serveral moviez taht now appear in "in progress movies". can i set the playback back? 

i now just configured the addon that it sync the playback after i watched ten minutes of the movies. but all the old files are stuck somewhere in 5 to 25 seconds.

hope you understand my problem.

**Comment by razzeee** [comment]
No, that's not possible. With kodi 18 you can reset your progress, but trakt doesn't honour it.

**Comment by xadox-1st** [comment]
@billystar1337 you could use https://github.com/sharkykh/TraktPlaybackProgressManager to remove the progress of specific movies or episodes from the trakt.tv db.

**Comment by sharkykh** [comment]
@xadox-1st Thank you for the shout out!
@billystar1337 
You can now use the web version over at [sharkykh.github.io/tppm](https://sharkykh.github.io/tppm/).
No need to install Python, and you can use it on mobile devices too 👌

**Comment by andre-koch** [comment]
Can this feature be added to the Trakt scrobbler?

**Comment by nekromantik** [comment]
this issue makes the sync playback progress useless as it does not reset it once you remove the status. :(
